Kernel/API/KResult.h

```diff
@@ -15,6 +15,19 @@
 
 namespace Kernel {
 
+constexpr StringView errno_to_string(ErrnoCode code)
+{
+    switch (code) {
+#define __ENUMERATE_ERRNO_CODE(c, s) \
+    case c:                          \
+        return s##sv;
+        ENUMERATE_ERRNO_CODES(__ENUMERATE_ERRNO_CODE)
```

Comment:
This relies on the system being serenity (you can't define your own errno codes everywhere, silly)

Reply:
(to be more clear, this would be fine if it wasn't included from LibCore)

Reply:
Hmm, good point, since we're also using this from the userland now... So I guess we could have this inside an `#ifdef __serenity__`, and fallback to `strerror` on non-Serenity platforms.